main.sh

```diff
@@ -100,6 +100,18 @@ download_and_extract() {
                 esac
             fi
             ;;
+        *.tar.zst | *.tzst)
+            tar_args+=("xvf" "--zstd")
```

Comment:
```suggestion
            tar_args+=("--zstd" "xvf")
```

(Argument after f (-f) is considered filename.)

Reply:
:facepalm: of course :D